File: MyWami/src/com/MyWami/model/UserModel.java
package com.MyWami.model;

/**
 * Created by robertlanter on 2/12/14.
 */
public class UserModel {
		private int userId;
		private String userName;
		private String passWord;

	public String getPassWord() {
		return passWord;
	}

	public void setPassWord(String passWord) {
		this.passWord = passWord;
	}

	public String getUserName() {
			return userName;
		}

		public void setUserName(String userName) {
			this.userName = userName;
		}

		public int getUserId() {
			return userId;
		}

		public void setUserId(int userId) {
			this.userId = userId;
		}
	}

File: MyWamiIOS/MyWami/MyWami/UserModel.swift
//
//  UserModel.swift
//  MyWami
//
//  Created by Robert Lanter on 4/6/15.
//  Copyright (c) 2015 Robert Lanter. All rights reserved.
//

import Foundation

public class UserModel {
    var userId: Int = 0
    var userName: String = ""
    var userPassword: String = ""

    func setUserId(userId: Int) {
        self.userId = userId
    }
    
    func getUserId() -> Int {
        return self.userId
    }
    
    func setUserName(userName: String) {
        self.userName = userName
    }
    
    func getUserName() -> String {
        return self.userName
    }
    
    func setUserPassword(userPassword: String) {
        self.userPassword = userPassword
    }
    
    func getUserPassword() -> String {
        return self.userPassword
    }
    
}
